src/integTest/resources/projects/standard-project-java8/src/main/java/HelloServlet.java

```diff
@@ -0,0 +1,19 @@
+import java.io.IOException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class HelloServlet extends HttpServlet {
+
+  @Override
+  public void service(HttpServletRequest req, HttpServletResponse resp) throws IOException {
+    resp.setContentType("text/plain");
+
+    // no cache
+    resp.setHeader("Cache-Control", "no-cache, no-store, must-revalidate"); // HTTP 1.1.
+    resp.setHeader("Pragma", "no-cache"); // HTTP 1.0.
+    resp.setHeader("Expires", "0"); // Proxies.
+
+    resp.getWriter().print("Hello from the App Engine Standard Java project.");
```

Comment:
The expected output is `Hello from the App Engine Standard project`, remove the `Java`

See https://github.com/GoogleCloudPlatform/app-gradle-plugin/blob/master/src/integTest/java/com/google/cloud/tools/gradle/appengine/AppEngineStandardPluginIntegrationTest.java#L70